datalad/distribution/create_publication_target_sshwebserver.py

```diff
@@ -249,29 +230,27 @@ def __call__(sshurl, target=None, target_dir=None,
                     else:
                         raise ValueError("Do not know how to hand existing=%s" % repr(existing))
 
-                cmd = ssh_cmd + ["mkdir", "-p", path]
                 try:
-                    runner.run(cmd)
+                    ssh(["mkdir", "-p", path])
                 except CommandError as e:
                     lgr.error("Remotely creating target directory failed at "
                               "%s.\nError: %s" % (path, str(e)))
                     continue
 
             # init git repo
-            cmd = ssh_cmd + ["git", "-C", path, "init"]
+            cmd = ["git", "-C", path, "init"]
             if shared:
                 cmd.append("--shared=%s" % shared)
             try:
-                runner.run(cmd)
+                ssh(cmd)
             except CommandError as e:
                 lgr.error("Remotely initializing git repository failed at %s."
                           "\nError: %s\nSkipping ..." % (path, str(e)))
                 continue
 
             # check git version on remote end:
-            cmd = ssh_cmd + ["git", "version"]
             try:
-                out, err = runner.run(cmd)
+                out, err = ssh(["git", "version"])
                 git_version = out.lstrip("git version").strip()
                 lgr.debug("Detected git version on server: %s" % git_version)
                 if git_version < "2.4":
```

Comment:
nah... versions shouldn't be compared as strings or sh.t will get loose when git becomes '10.0'... use e.g. LooseVersion... e.g. sufficient to wrap git_version into a LooseVersion object:

```shell
$> python -c 'from distutils.version import LooseVersion; print LooseVersion("2.4") < "2.5"'
True

$> python -c 'from distutils.version import LooseVersion; print LooseVersion("2.4") < "2.2"'
False
```